the-collab-lab/tcl-28-smart-shopping-list
# Ks radio styling

## Description


- Re-styled existing frequency radio inputs in AddItem component to look and act like buttons
- Incorporated new color palette
- Added `:focus` and `:checked` styling features 

## Related Issue


closes #48 

## Acceptance Criteria


- [x] Make radio inputs look like buttons
- [x] Maintain existing functionality 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |

## Updates

### Before


![Screen Shot 2021-08-31 at 12 51 37 PM](https://user-images.githubusercontent.com/40218057/131559568-7f26f3e9-cf9a-447b-bd2f-05cfc67610a7.png)

### After


![Screen Shot 2021-08-31 at 12 59 31 PM](https://user-images.githubusercontent.com/40218057/131560609-b891a107-4808-44ee-ae92-ddb7083f4925.png)

## Testing Steps / QA Criteria


- checkout `ks-radio-styling` branch
- navigate to Add an Item
- Try to add an item without selecting a frequency, you should still get an error
- Add an item with a frequency selected, it should appear on your list with the proper frequency
- Navigate through frequencies using proper keys instead of mouse (tab to get to the first frequency, spacebar to select, up/down keys to move between them) and all should still function as expected
